pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/CloseResource.xml

```diff
@@ -377,7 +377,7 @@ public class Test {
 
     <test-code>
         <description>#947 CloseResource rule fails if field is marked with annotation</description>
-        <expected-problems>2</expected-problems>
+        <expected-problems>3</expected-problems>
```

Comment:
Ok, we have the following reported problems:

n/a:8:	Ensure that resources like this Connection object are closed after use
n/a:9:	Ensure that resources like this Statement object are closed after use
n/a:13:	'rs' is not closed within a finally block, thus might not be closed at all in case of exceptions

so that seems ok.